### ISSUE TITLE
Select Midi Channel per control

### ISSUE THREAD

**Issue by wchorski** [shaltiel/42Noir-UnityEasyController#1]
I have a midi controller that has a few controls mapped to the same cc but on different channels. Example

- Fader A - 28cc | midi channel 1
- Fader B - 28cc | midi channel 2

How can I isolate control when mapping inside EasyController? As of now when I map either fader on my controller, they both control the one knob in EasyController

**Comment by shaltiel** [comment]
We use all channels for simplicity, that’s why there is no difference between faders in different channels. It is restrictive in one way but remove some uncertainty in live applications

**Comment by antiero** [comment]
Yeah this would be super handy when using an OP-Z, using it's different track (which by default output to different MIDI channels) to control different stuff.

**Comment by shaltiel** [comment]
Would it be sufficient to send the channel info to unity? So you can use it to define behaviour according to channel? That I can add, but changing the knob assignment with changing the channel is probably too much work in that stage…

**Comment by wchorski** [comment]
sending channel info to Unity wouldn't achieve the intent. Any way to create pages (in this apps UI) like changing banks? Like how in Ableton I have a drop down of what MIDI channel I am using

![image](https://user-images.githubusercontent.com/33769609/235306317-a2aa7873-39e5-452c-9993-83c88fcd5a9f.png)

If that isn't possible to have different pages, each one with it's own set of knobs, maybe have a filter at the singular knob level. Can a knob just listen to a single midi channel i.e.

- s1a -> s8a = midi channel 1
- s1b -> s8b = midi channel 13
- s1c -> s8c = midi channel all

**Comment by shaltiel** [comment]
Ok let me check what can be done it should be automatic though without the need to set anything by the user..
